Stripe/STPRedirectClient.h

```diff
@@ -0,0 +1,61 @@
+//
+//  STPRedirectClient.h
+//  Stripe
+//
+//  Created by Brian Dorfman on 1/23/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+#import "STPSource.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class STPSource;
+@class STPRedirectConfiguration;
+
+typedef void (^STPRedirectAuthCompletionBlock)(STPSource * __nullable source, NSError * __nullable error);
+
+@interface STPRedirectClient : NSObject
+@property (nonatomic, readonly) STPRedirectConfiguration *configuration;
+@property (nonatomic, nullable, readonly) STPSource *inProgressAuthSource;
+
+- (instancetype)init NS_UNAVAILABLE;
+- (instancetype)initWithCoder:(NSCoder *)aDecoder NS_UNAVAILABLE;
+- (instancetype)initWithConfiguration:(STPRedirectConfiguration *)configuration NS_EXTENSION_UNAVAILABLE("Redirect based sources are not available in extensions");
+
+/**
+ Starts a new redirect flow authorization for the given source.
```

Comment:
Don't see a way to change the Xcode behavior, and I would rather standardize around the way the official tools do it if thats ok.